### PR TITLE
Feature/land registry

### PR DESCRIPTION
**Pull request by saniemi**
A few minor changes to address codacy issues and some earlier PR comments. No changes in performance or functionality.

### CODE REVIEW

File: DataScience/Analytics/data/landRegistryData.py [hunk 1]
```diff
@@ -1,3 +1,36 @@
+"""
+ONS Address Index - Land Registry Data
+======================================
+
+A simple script to process land registry sales data.
```
Comment (analytically) [inline DataScience/Analytics/data/landRegistryData.py]: mind explaining how to run it, what params etc? give some examples

File: DataScience/Analytics/data/landRegistryData.py [hunk 2]
```diff
@@ -1,3 +1,36 @@
+"""
```
Comment (analytically) [inline DataScience/Analytics/data/landRegistryData.py]: mind starting the file with `#!/usr/bin/env python`

Comment (saniemi) [inline DataScience/Analytics/data/landRegistryData.py]: Added.

**Comment by Mironor** [comment]
LGTM

**Pull request merged**